sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-data-inventory-report.sql

```diff
@@ -0,0 +1,527 @@
+            parent1_users.last_name as "Parent 1 last name",
+            parent1.contact_info -> ''address'' ->> ''postalCode'' as "Parent 1 postal code",
+            parent1.supporting_data ->> ''totalIncome'' as "Parent 1 income (line 15000)",
+            current_assessment.workflow_data -> ''calculatedData'' ->> ''totalParentNetIncome'' as "Parent 1 net annual income (calculated value)",
+            parent1.supporting_data ->> ''cppLine30800'' as "Parent 1 Canada pension plan contribution (line 30800)",
+            parent1.supporting_data ->> ''cppLine31000'' as "Parent 1 Canada pension plan contribution (line 31000)",
+            parent1.supporting_data ->> ''eiLine31200'' as "Parent 1 Employment Insurance Premium (Line 31200)",
+            parent1.supporting_data ->> ''totalIncomeTaxLine43500'' as "Parent 1 Income tax paid (Line 43500)",
+            parent1.supporting_data ->> ''parentalContributions'' as "Parent 1 contribution to student",
+            parent1.supporting_data ->> ''parentsOtherDependants'' as "Parent 1 dependants other than applicant",
+            jsonb_array_length(parent1.supporting_data -> ''parentDependentTable'') as "Parent 1 number of other dependants",
+            parent1_dependant_number_post_sec_school.count as "Parent 1 number of dependants on post secondary school",
+            parent1_dependant_number_disability.count as "Parent 1 number of dependants with disability",
+            -- Parent2 Info
+            parent2_users.first_name as "Parent 2 first name",
+            parent2_users.last_name as "Parent 2 last name",
+            parent2.contact_info -> ''address'' ->> ''postalCode'' as "Parent 2 postal code",
+            parent2.supporting_data ->> ''totalIncome'' as "Parent 2 income (line 15000)",
+            current_assessment.workflow_data -> ''calculatedData'' ->> ''totalParentNetIncome'' as "Parent 2 net annual income (calculated value)",
+            parent2.supporting_data ->> ''cppLine30800'' as "Parent 2 Canada pension plan contribution (line 30800)",
+            parent2.supporting_data ->> ''cppLine31000'' as "Parent 2 Canada pension plan contribution (line 31000)",
+            parent2.supporting_data ->> ''eiLine31200'' as "Parent 2 Employment Insurance Premium (Line 31200)",
+            parent2.supporting_data ->> ''totalIncomeTaxLine43500'' as "Parent 2 Income tax paid (Line 43500)",
+            parent2.supporting_data ->> ''parentalContributions'' as "Parent 2 contribution to student",
+            parent2.supporting_data ->> ''parentsOtherDependants'' as "Parent 2 dependants other than applicant",
+            jsonb_array_length(parent1.supporting_data -> ''parentDependentTable'') as "Parent 2 number of other dependants",
+            parent2_dependant_number_post_sec_school.count as "Parent 2 number of dependants on post secondary school",
+            parent2_dependant_number_disability.count as "Parent 2 number of dependants with disability",
+            -- Spouse Info
```

Comment:
@HRAGANBC @CarlyCotton @Joshua-Lakusta I believe that we started to use "Partner" instead "Spouse" and it must be followed everywhere, right?
If the spreadsheet has it as a "Spouse" I would say that it should be updated also. Does it make sense?
@andrepestana-aot this is not a blocker but something that may need to be adjusted for upcoming efforts if business confirms the convention.

Reply:
I removed from the code and updated the spreadsheet.